package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/rollup-plugin-md.js",
   "jsnext:main": "dist/rollup-plugin-md.mjs",
   "scripts": {
-    "test": "mocha test/*.js --compilers js:buble/register",
+    "test": "mocha test/*.js",
```

Comment:
The --compilers option is deprecated, and buble/register no longer works. Removed since it is no longer needed.